server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/dto/TaskIdDto.java

```diff
@@ -33,7 +33,7 @@
 
     public static TaskIdDto respond(Response response, TaskId taskId) {
         response.status(HttpStatus.CREATED_201);
-        response.header(LOCATION.asString(), TasksRoutes.BASE + "/" + taskId.toString());
+        response.header(LOCATION.asString(), TasksRoutes.BASE + "/" + taskId.getValue());
```

Comment:
does it fix anything ? I don't see any test change. Or maybe toString returns the same thing than getValue ?

Reply:
Yes it fixes something. Gonna write a test.